packages/angular_devkit/build_angular/src/webpack/configs/styles.ts

```diff
@@ -263,154 +266,33 @@ export function getStylesConfig(wco: WebpackConfigOptions): webpack.Configuratio
       },
     },
     {
-      loader: require.resolve('postcss-loader'),
+      loader: postCssLoaderPath,
       options: {
-        implementation: require('postcss'),
+        implementation: postCss,
         postcssOptions: postcssOptionsCreator(false, buildOptions.extractCss),
         sourceMap: !!cssSourceMap,
       },
     },
   ];
 
-  const styleLanguages: {
```

Comment:
Can we keep this data structure?  I wanted to eventually elevate it to an export and reuse the stylesheet language information in other areas (e.g., the Angular plugin's inline style config).

Reply:
done.